Update npm packages, fix high level security issues

theogravity opened the pull request:
This addresses the high level issues that `npm audit` generates. I was able to update all packages to current except for `marked`, which I was able to update to `0.3.12`, where all tests would still pass (the latest is `0.6.0`, it doesn't seem to follow semver convention, but it's around 900 commits later).

I've removed the node `0.10` / `0.12` builds from the travis config as they're extremely outdated node versions that are most likely not in use nowadays.

theogravity committed:
Update npm packages, fix high level security issues

theogravity committed:
Remove node 0.10 and 0.12 support

theogravity committed:
Update readme

theogravity committed:
Update dependencies

theogravity committed:
Update deps to fix npm audit

theogravity committed:
Update libs to latest

chocolateboy commented:
It looks like this has been [published](https://www.npmjs.com/package/toc-md-alt) (9 months ago).

Maybe close this (and update the repo)? Otherwise, it looks like this is still an issue.

chocolateboy committed:
Fix repo links

Link to the correct/current repo so issues aren't raised on the old repo.

theogravity committed:
Merge pull request #7 from chocolateboy/patch-1

Fix repo links

chocolateboy committed:
Add a note about migrating from other TOC tools

The README only mentions `<!-- TOC -->`, but `<!-- TOC END -->` is also needed when migrating from another TOC tool (with different markers) and updating.

theogravity committed:
Merge pull request #8 from chocolateboy/patch-2

Add a note about migrating from other TOC tools

chocolateboy committed:
consistently space the opening and closing comments

add an extra newline between the opening comment and the start of the TOC.

this makes the spacing for the opening comment consistent with the
closing comment, and is consistent with other tools, e.g. doctcoc and
markdown-toc (which preserves newlines)

theogravity committed:
Merge pull request #9 from chocolateboy/consistent-spacing

Consistently space the opening and closing comments

theogravity committed:
Update packages

theogravity committed:
Update changelog

theogravity committed:
Update changelog / package.json

theogravity committed:
Add circleci deploy

theogravity committed:
Add status badge

theogravity committed:
Add CONTRIBUTING.md

theogravity committed:
Add CI support

theogravity committed:
0.4.1 [skip ci]

chocolateboy committed:
Bump dependencies (#11)

+ unpin versions

theogravity committed:
0.4.2 [skip ci]
